packages/flutter_tools/test/general.shard/android/kotlin_test.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
what about naming this file `template_test.dart`, so folks can find the tests for the file `template.dart`